css/src/components/hero.scss

```diff
@@ -30,12 +30,15 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 			width: $hero-content-fluid-width;
 			min-height: $hero-min-height-default;
 			padding-block: $hero-padding-default;
-			padding-inline-end: $hero-padding-sm;
+			// not using logical properties here to avoid spacing issues for RTL
+			padding-right: $hero-padding-sm; /* stylelint-disable-line csstools/use-logical */
 		}
 	}
 
 	&:not(.hero-image) {
 		.hero-content {
+			margin: auto;
```

Comment:
Can you explain the reasoning behind this update please? Looks like it changes the positioning of content:

![image](https://github.com/microsoft/atlas-design/assets/57374379/694d93d9-c8b6-4897-87b9-c1b611935139)

Reply:
Aka this:

![image](https://github.com/microsoft/atlas-design/assets/57374379/8e2d91c4-b1eb-4071-9371-1fbd0468abb9)

Reply:
It was originally suggested in an earlier comment. I removed it since there isn't any extra left spacing for LTR.